File: Java/Basics To Java/src/D/D6.java
package D;

import java.util.Random;
import java.util.Scanner;

public class D6 {
	public static void mainx(String[] args) {
		Scanner sc = new Scanner(System.in);
		Random r = new Random();
		int odhad;
		int nahoda;

		do {
			nahoda = r.nextInt(1, 11);
			System.out.println("Hádej co padlo..");
			odhad = sc.nextInt();

			if (odhad == nahoda)
				System.out.println("TREFA!");
			else
				System.out.println("špatnì!");
		} while (odhad != nahoda);
		sc.close();
	}
}

File: C#/Basic logic/D/D6.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace D
{
    internal class D6
    {
        static void Mainx(string[] args)
        {
            Random r = new Random();
            int odhad;
            int nahoda;

            do
            {
                nahoda = r.Next(1, 11);
                Console.WriteLine("Hádej co padlo..");
                odhad = int.Parse(Console.ReadLine());

                if (odhad == nahoda)
                    Console.WriteLine("TREFA!");
                else
                    Console.WriteLine("špatně!");
            }
            while (odhad != nahoda);
        }
    }
}